include/camp/helpers.hpp

```diff
@@ -79,6 +79,37 @@ namespace type
     using add = T&&;
   }  // end namespace rvref
 
+  namespace ptr
+  {
+    template <class T>
+    struct rem_s {
+      using type = T;
+    };
+    template <class T>
+    struct rem_s<T*> {
+      using type = T;
+    };
+    template <class T>
```

Comment:
This specialization needs to come out, or it will eat a double pointer down to a base type in one step, which I don't think is what we want.

Reply:
I see, there could be other qualifiers in between. Done.